### PR TITLE
docs: update homepage initial docs

### PR DESCRIPTION
**Pull request by matyasf**

### CODE REVIEW

File: docs/getting-started/dev-commands.md [hunk 1]
```diff
@@ -5,22 +5,23 @@ order: 3
...
+| `yarn test:watch --scope <package-name>`  | Identical to `yarn test:watch` but scoped to a single package. For example, to run the command only for the `ui-avatar` package you would do `yarn test:watch --scope @instructure/ui-avatar`                                                                                                                                               |
+| `yarn start:examples`                     | Starts a production instance of [storybook](https://storybook.js.org/) with all component examples.                                                                                                                                                                                                                                         |
+| `yarn dev:examples`                       | Starts a dev instance of [storybook](https://storybook.js.org/) with all component examples.                                                                                                                                                                                                                                                |
```
Comment (matyasf) [inline docs/getting-started/dev-commands.md:25]: this file got autoformatted, this is the real change here

File: docs/getting-started/releases.md [hunk 2]
```diff
@@ -25,64 +26,3 @@ Every commit that has functional changes to the `master` branch constitutes as a
 1. Verify that the release was [tagged in Github](https://github.com/instructure/instructure-ui/releases).
 1. Verify that the release was published to npm by running `yarn info [package]@[version]`.
 1. Verify that the [documentation was updated](https://instructure.design/).
-
-__To manually run the release:__
-
```
Comment (matyasf) [inline docs/getting-started/releases.md:30]: This part needs to be rewritten when @HerrTopi 's work is complete

File: docs/newsletter.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 ---
-title: Newsletter
+title: Newsletter (old)
 parent: index
 id: Newsletter
 ---
 
+> Note: For the latest updated please check the [changelog](#CHANGELOG)
+
```
Comment (matyasf) [inline docs/newsletter.md:8]: I've decided to deprecate this document, we already have a changelog and will make a 8.0 upgrade guide. But if someone feels strongly that we should be releasing updates here too, then lets discuss it.

Comment (matyasf) [reply docs/newsletter.md:8]: the changes after this are from the autoformatter.

**Review by HerrTopi — APPROVED** [review]

**Review by ke1k0 — APPROVED** [review]

**Pull request merged**